components/form-builder/app/Publish.tsx

```diff
@@ -85,83 +87,102 @@ export const Publish = () => {
-              <Markdown options={{ forceBlock: true }}>
-                {t("contactSupportIfYouHaveQuestions")}
-              </Markdown>
-            </div>
-          </div>
-          <Button className="mt-5" onClick={handlePublish}>
-            {t("publish")}
-          </Button>
-          <div
-            role="alert"
-            className={`inline-block ml-5 py-1 px-3 
+          )}
+
+          <ul className="list-none p-0">
+            <li className="mb-4 mt-4">
+              <Icon checked={title} />
+              <Link href={`/${i18n.language}/form-builder/edit#formTitle`}>{t("formTitle")}</Link>
+            </li>
+            <li className="mb-4 mt-4">
+              <Icon checked={questions} />
+              <Link href={`/${i18n.language}/form-builder/edit`}>{t("questions")}</Link>
+            </li>
+            <li className="mb-4 mt-4">
+              <Icon checked={privacyPolicy} />
+              <Link href={`/${i18n.language}/form-builder/edit#privacy-text`}>
+                {t("privacyStatement")}
+              </Link>
+            </li>
+            <li className="mb-4 mt-4">
+              <Icon checked={confirmationMessage} />
+              <Link href={`/${i18n.language}/form-builder/edit#confirmation-text`}>
+                {t("formConfirmationMessage")}
+              </Link>
+            </li>
+            <li className="mb-4 mt-4">
+              <Icon checked={translate} />
+              <Link href={`/${i18n.language}/form-builder/edit/translate`}>{t("translate")}</Link>
+            </li>
+          </ul>
+
+          {userCanPublish && isPublishable() && (
+            <>
+              <Button className="mt-5" onClick={handlePublish}>
+                {t("publish")}
+              </Button>
+              <div
+                role="alert"
+                className={`inline-block ml-5 py-1 px-3 
             ${error ? "text-red-destructive bg-red-100" : ""}
             ${!error ? "hidden" : ""}`}
-          >
-            <>{error && <p>{t("thereWasAnErrorPublishing")}</p>}</>
-          </div>
-        </>
-      )}
+              >
+                <>{error && <p>{t("thereWasAnErrorPublishing")}</p>}</>
+              </div>
+            </>
+          )}
+        </div>
+        {userCanPublish && isPublishable() && (
+          <>
+            <div className="flex-none max-w-md">
+              <InfoCard title={t("whatYouNeedToKnow")}>
+                <div className="mb-5 bg-gray-50 p-1.5">
+                  <p className="mb-1 font-bold pb-0">{t("publishingDisablesEditing")}</p>
+                  <p className="text-sm">{t("publishingDisablesEditingDescription")}</p>
+                </div>
+                <div className="mb-5 bg-gray-50 p-1.5">
+                  <p className="mb-1 font-bold pb-0">{t("publishingLocksSettings")}</p>
+                  <p className="text-sm">{t("publishingLocksSettingsDescription")}</p>
+                </div>
+                {isVaultDelivery() && (
```

Comment:
Looks like your not passing the delivery option here so this check is not working when email delivery is set.

Reply:
fixed